core/deployment/src/main/java/io/quarkus/deployment/dev/QuarkusCompiler.java

```diff
@@ -170,7 +174,7 @@ public void setupSourceCompilationContext(DevModeContext context, Set<File> clas
                                 sourcePath.toFile(),
                                 new File(compilationUnit.getClassesPath()),
                                 context.getSourceEncoding(),
-                                context.getCompilerOptions(),
+                                context.getCompilersOptions(),
```

Comment:
Actually, let's revert this one. `compilerOptions` is proper English.

Reply:
ok, I will revert it everywhere